daprdocs/content/en/reference/components-reference/supported-bindings/postgresql.md

```diff
@@ -22,10 +22,9 @@ spec:
   type: bindings.postgresql
   version: v1
   metadata:
-  - name: url # Required
-    value: "<CONNECTION_STRING>"
-  - name: direction
-    value: "<DIRECTION_OF_BINDING>"
```

Comment:
Please include the DIRECTION_OF_BINDING

Reply:
This is only an output binding so the direction is irrelevant. I think it causes more confusion than anything.